tb_plugin/torch_tb_profiler/consts.py

```diff
@@ -1,17 +1,16 @@
 # -------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # --------------------------------------------------------------------------
-
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
+import re
 from collections import namedtuple
 
 PLUGIN_NAME = "pytorch_profiler"
 
-TRACE_FILE_SUFFIX = ".pt.trace.json"
-TRACE_GZIP_FILE_SUFFIX = ".pt.trace.json.gz"
+WORKER_PATTERN = re.compile(r"""^(.*?) # worker name
+        # TODO: uncomment the following line when we need supprort multiple steps
+        # (?:\.\d+)? # optional timestamp like 1619499959628
+        \.pt\.trace\.json # the ending suffix
+        (?:\.gz)?$""", re.X)  # optional .gz extension
```

Comment:
the timestamp is optional because user might provide the file name without calling the [tensorboard_trace_handler](https://github.com/pytorch/pytorch/blob/master/torch/profiler/profiler.py#L58), We would add multiple step in next PR which will need the timestamp information. If there is not timestamp, which means that the file name is specified manually by user which is not multiple steps scenarios.

Reply:
But if it's not going to be optional, then this regex will be buggy (since it will still accept an input without the timestamp). I think you should either remove the timestamp from the regex, or make it required by replacing the `?`.

Reply:
you can refer to https://github.com/pytorch/kineto/pull/215 for how the timestamp is used.

Reply:
Makes sense, I can see you're using the capturing groups directly so it does seem better to do it this way. I still think it would be nice to comment an example name, showing what we expect the data to look like, e.g. `anything-here.123.pt.trace.json.gz`